definitions/tools/multiqc.cwl

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "multiqc"
+baseCommand: ["/usr/local/bin/multiqc"]
+requirements:
+    - class: ShellCommandRequirement
+    - class: ResourceRequirement
+      ramMin: 4000  
+    - class: DockerRequirement
+      dockerPull: "sridnona/multiqc:1.9"
```

Comment:
We're doing our best to move away from docker hosted repos outside mgibio (since they'll be rate-limited for outside folks in Nov).  Thoughts on either finding a quay-hosted repo or moving this into mgibio?

Reply:
will move this to mgibio repo.